database/dbMethods.js

```diff
@@ -1,38 +1,34 @@
-const client = require('../database/index');
+/* eslint-disable no-console */
+const client = require('./index');
 
 module.exports = { 
   getOne: (selector) => {
-    return client.query('select * from stockinfo where stockid=$1', [selector])
+    return client.query(
+      'select * from stockinfo where stockid=$1',
+      [selector])
     .then((res) => {
       return res.rows[0];
     })
     .catch(e => console.error(e.stack));
```

Comment:
just a question for my knowledge - does e.stack return some more useful info that I should be aware of?